types/types.go

```diff
@@ -0,0 +1,12 @@
+package types
+
+import (
+	"github.com/bmc-toolbox/common"
+)
+
+type BiosConfig map[string]string
+
+type ComponentInventoryDevice struct {
```

Comment:
Consider renaming this to `ComponentInventory` or `InventoryDevice` or `DeviceInventory`? Something a little less long and more targeted.